lib/appSocketToClient.ts

```diff
@@ -57,6 +58,7 @@ export function requestBroadcastMatch(matchId: string) {
   for (const url in global.appSocketToWebSocketServer) {
     if (global.appSocketToWebSocketServer[url].connected) {
       global.appSocketToWebSocketServer[url]?.emit('broadcastMatch', matchId);
```

Comment:
Some refactoring we can do if my understanding is correct:

- Change `global.appSocketToWebSocketServer` back to just a `Socket` instead of a dictionary
- Change `WEBSOCKET_URLS` to `WEBSOCKET_URL` (only need the socket of the current server)
- Update all of these request functions to just be:

```
    if (global.appSocketToWebSocketServer?.connected) {
      global.appSocketToWebSocketServer.emit('broadcastMatches');
    }
```